eas.json

```diff
@@ -0,0 +1,24 @@
+{
+  "cli": {
+    "version": ">= 7.8.1"
+  },
+  "build": {
+    "preview": {
+      "android": {
+        "buildType": "apk"
+      }
+    },
+    "preview2": {
+      "android": {
+        "gradleCommand": ":app:assembleRelease"
+      }
+    },
+    "preview3": {
+      "developmentClient": true
+    },
+    "preview4": {
+      "distribution": "internal"
```

Comment:
No need to have 4 preview build profiles. The two that we will need are `development` and `production` - one for us to test the app and one for the releases and milestones.